api/v1alpha1/imageclusterinstall_types.go

```diff
@@ -100,6 +100,11 @@ type ImageClusterInstallSpec struct {
 	// Proxy defines the proxy settings to be applied in relocated cluster
 	// +optional
 	Proxy *Proxy `json:"proxy,omitempty"`
+
+	// AdditionalNTPSources is a list of NTP sources (hostname or IP) to be added to all cluster
+	// hosts. They are added to any NTP sources that were configured through other means.
+	// +optional
+	AdditionalNTPSources []string `json:"additionalNTPSources,omitempty"`
```

Comment:
:+1: 
I didn't know we (does recert?) support this

Reply:
Yeah, we support if https://github.com/openshift-kni/lifecycle-agent/blob/main/api/seedreconfig/seedreconfig.go#L147